Bottom: 314a187bbc4e102fefaed4583b0ce919d7fb9a42
Top:    850e2a96d4e7041c6a150a09fd07ddbee58bcca4
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-20 11:10:56 +0800

net: sock: introduce SOCK_XDP


---

diff --git a/include/net/sock.h b/include/net/sock.h
index 83b7475..c8cc8e3 100644
--- a/include/net/sock.h
+++ b/include/net/sock.h
@@ -800,6 +800,7 @@ enum sock_flags {
 	SOCK_SELECT_ERR_QUEUE, /* Wake select on error queue */
 	SOCK_RCU_FREE, /* wait rcu grace period in sk_destruct() */
 	SOCK_TXTIME,
+	SOCK_XDP, /* XDP is attached */
 };
 
 #define SK_FLAGS_TIMESTAMP ((1UL << SOCK_TIMESTAMP) | (1UL << SOCK_TIMESTAMPING_RX_SOFTWARE))
